README.md

```diff
@@ -2023,12 +2023,26 @@ end
 
 Built-in formatters are the following.
 
-* `:json`: use object's `to_json` when available, otherwise call `MultiJson.dump`
+* `:json`: use object's `to_json` when available, otherwise call `JSON.dump`
 * `:xml`: use object's `to_xml` when available, usually via `MultiXml`, otherwise call `to_s`
 * `:txt`: use object's `to_txt` when available, otherwise `to_s`
 * `:serializable_hash`: use object's `serializable_hash` when available, otherwise fallback to `:json`
 * `:binary`: data will be returned "as is"
 
+Grape now not longer use MultiJson as JSON processor.If you want to continue to use MultiJson, you can use it by below
```

Comment:
This should be moved to UPGRADING. Some minor "better" English: "Grape no longer uses MultiJson as default JSON processor. To use MultiJson add `multi_json` to your Gemfile and set `config.json_processor`.